adagucserverEC/CDBAdapterPostgreSQL.cpp

```diff
@@ -1080,3 +1080,87 @@ int CDBAdapterPostgreSQL::addFilesToDataBase() {
 #endif
   return 0;
 }
+
+int CDBAdapterPostgreSQL::storeLayerMetadata(const char *datasetName, const char *layerName, const char *metadataKey, const char *metadataBlob) {
+#ifdef MEASURETIME
+  StopWatch_Stop(">CDBAdapterPostgreSQL::storeLayerMetadata");
+#endif
+  CPGSQLDB *dataBaseConnection = getDataBaseConnection();
+  if (dataBaseConnection == NULL) {
+    return -1;
```

Comment:
Maybe we could do `CDBError("something something"); throw(__LINE__);` here as well? For debugging purposes in the future.

Reply:
Good point. Lets keep it in mind. Until now it did not cause issues as the -1 one is captured at a later stage.

Reply:
Ah, perfect!